src/main.c

```diff
@@ -10,7 +10,11 @@
 #undef _POSIX_C_SOURCE
 #define _POSIX_C_SOURCE 200809L
 #undef _XOPEN_SOURCE
+#ifdef IS_SOLARIS	/* Solaris doesn't recognize this macro greater than 600 */
```

Comment:
if that is so, then i suggest we add -D_XOPEN_SOURCE to CFLAGS and here:
```
#ifndef _XOPEN_SOURCE
#define _XOPEN_SOURCE 700
#endif

Reply:
Does the `-std=c99` is required? If so I will trying to add this option to `configure`